### ISSUE TITLE
Crash when querying or inserting to a table

### ISSUE THREAD

**Issue by shobhik** [Raizlabs/DBFlow#1373]
DBFlow Version: 4.0.5
Issue Kind: Bug

Description: 

I am using DBFlow 4.0.5 with SQLCipher. Anytime I try to either query or insert an object in a table that I know exists, I get an AbstractMethodError exception.

Query exception:
```	java.lang.AbstractMethodError: abstract method "com.raizlabs.android.dbflow.structure.database.FlowCursor com.raizlabs.android.dbflow.structure.database.DatabaseWrapper.rawQuery(java.lang.String, java.lang.String[])"
		at com.raizlabs.android.dbflow.sql.queriable.ModelLoader.load(ModelLoader.java:59)
		at com.raizlabs.android.dbflow.sql.queriable.ListModelLoader.load(ListModelLoader.java:44)
		at com.raizlabs.android.dbflow.sql.queriable.ListModelLoader.load(ListModelLoader.java:15)
		at com.raizlabs.android.dbflow.sql.queriable.ModelLoader.load(ModelLoader.java:53)
		at com.raizlabs.android.dbflow.sql.queriable.ListModelLoader.load(ListModelLoader.java:37)
		at com.raizlabs.android.dbflow.sql.queriable.ListModelLoader.load(ListModelLoader.java:15)
		at com.raizlabs.android.dbflow.sql.queriable.ModelLoader.load(ModelLoader.java:36)
		at com.raizlabs.android.dbflow.sql.queriable.ListModelLoader.load(ListModelLoader.java:25)
		at com.raizlabs.android.dbflow.sql.language.BaseModelQueriable.queryList(BaseModelQueriable.java:61)
		at com.raizlabs.android.dbflow.sql.language.Where.queryList(Where.java:236)
		at com.shobhik.example.app.database.DatabaseHelper.getMyObjectWithStatus(DatabaseHelper.java:33)
		at com.shobhik.example.app.selectors.MyObjectSelector.getMyObjectWithStatus(MyObjectSelector.java:25)
		at com.shobhik.example.commons.Object.ObjectSelector$LoadDataTask.doInBackground(ObjectSelector.java:234)
		at com.shobhik.example.commons.Object.ObjectSelector$LoadDataTask.doInBackground(ObjectSelector.java:222)
		at android.os.AsyncTask$2.call(AsyncTask.java:292)
		at java.util.concurrent.FutureTask.run(FutureTask.java:237)
		at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
		at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
		at java.lang.Thread.run(Thread.java:818)
```
Insert Exception:
```	  java.lang.RuntimeException: An error occured while executing doInBackground()
		  at android.os.AsyncTask$3.done(AsyncTask.java:304)
		  at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
		  at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
		  at java.util.concurrent.FutureTask.run(FutureTask.java:242)
		  at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
		  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
		  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
		  at java.lang.Thread.run(Thread.java:818)
	   Caused by: java.lang.AbstractMethodError: abstract method "void com.raizlabs.android.dbflow.structure.database.DatabaseStatement.bindStringOrNull(int, java.lang.String)"
		  at com.shobhik.example.app.model.core.MyObject_Table.bindToInsertStatement(MyObject_Table.java:161)
		  at com.shobhik.example.app.model.core.MyObject_Table.bindToInsertStatement(MyObject_Table.java:30)
		  at com.raizlabs.android.dbflow.structure.ModelAdapter.bindToInsertStatement(ModelAdapter.java:249)
		  at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.insert(ModelSaver.java:113)
		  at com.raizlabs.android.dbflow.sql.saveable.ListModelSaver.insertAll(ListModelSaver.java:54)
		  at com.raizlabs.android.dbflow.structure.ModelAdapter.insertAll(ModelAdapter.java:204)
		  at com.raizlabs.android.dbflow.structure.database.transaction.FastStoreModelTransaction$2.processModel(FastStoreModelTransaction.java:36)
		  at com.raizlabs.android.dbflow.structure.database.transaction.FastStoreModelTransaction.execute(FastStoreModelTransaction.java:88)
		  at com.raizlabs.android.dbflow.config.DatabaseDefinition.executeTransaction(DatabaseDefinition.java:280)
		  at com.shobhik.example.commons.Object.ObjectManager$ObjectsTask.doInBackground(ObjectManager.java:240)
		  at com.shobhik.example.commons.Object.ObjectManager$ObjectsTask.doInBackground(ObjectManager.java:218)
		  at android.os.AsyncTask$2.call(AsyncTask.java:292)
		  at java.util.concurrent.FutureTask.run(FutureTask.java:237)
		  at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231) 
		  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
		  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
		  at java.lang.Thread.run(Thread.java:818) 
```

**Comment by agrosner** [comment]
looks like you must be referencing the wrong version of DBFlow somewhere. ensure the processor, dbflow and sqlcipher packages all match version.

**Comment by shobhik** [comment]
@agrosner I think this might be tied to #1264. My workaround there was to use sqlcipher version 4.0.0-beta5.

**Comment by agrosner** [comment]
are you using proper versions? they should all match same version.

**Comment by shobhik** [comment]
@agrosner that was indeed what was happening. I just solved my issue in #1264 and now that all dependencies are on the same version, this issue looks to be resolved. Thanks for the help!
